config/tests/guest/libvirt/backuprestore.cfg

```diff
@@ -0,0 +1,214 @@
+        no virsh.snapshot_disk.no_delete.negative_test.pool_vol.disk_pool.attach_img_qed.snapshot_default.revert_paused
+        #gluster not supported on HOST OS
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.multi_disks.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_external
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.multi_disks.snapshot_from_xml.disk_external.disk_only.memory_none
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.transport_default.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_external
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.transport_default.snapshot_from_xml.disk_external.disk_only.memory_none
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.transport_rdma.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_external
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.transport_rdma.snapshot_from_xml.disk_external.disk_only.memory_none
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.transport_tcp.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_external
+        no virsh.snapshot_disk.no_delete.positive_test.no_pool.network_disk.gluster.transport_tcp.snapshot_from_xml.disk_external.disk_only.memory_none
+        no virsh.snapshot_disk.no_delete.negative_test.no_pool.network_disk.inactive.snapshot_from_xml.disk_internal.memory_internal
+        no virsh.snapshot_disk.no_delete.negative_test.no_pool.network_disk.inactive.snapshot_from_xml.disk_internal.memory_none
+        no virsh.snapshot_disk.no_delete.negative_test.no_pool.network_disk.inactive.snapshot_from_xml.disk_external.sys_checkpoint.default.memory_internal
+        no virsh.snapshot_disk.no_delete.negative_test.no_pool.network_disk.inactive.snapshot_from_xml.disk_external.disk_only.memory_none
+        #Skipping test case due to bug (https://bugzilla.redhat.com/show_bug.cgi?id=1083345)
+        no virsh.snapshot_disk.delete_test.positive_test.no_pool.attach_img_qcow2.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.dir_pool.attach_img_qcow2.v_qcow2.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.dir_pool.attach_img_qcow2.v_qcow2v3.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.netfs_pool.attach_img_qcow2.v_qcow2.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.netfs_pool.attach_img_qcow2.v_qcow2v3.snapshot_from_xml.disk_internal.memory_no
+        no virsh.snapshot_disk.delete_test.positive_test.pool_vol.netfs_gluster_pool.attach_img_qcow2.v_qcow2.snapshot_from_xml.disk_internal.memory_no
```

Comment:
same here,  something like this should cover all `memory_no` tests, choose according to your needs, `virsh.snapshot_disk..memory_no`